src/chapters/forms/checkbox-input.rst

```diff
@@ -0,0 +1,84 @@
+Checkbox Input
+==============
+
+A checkbox input represents a box to check. Checkbox inputs can be
+used by themselves or in groups. Checkbox inputs are best used with ``<label>`` tags.
+
+.. role:: raw-html(raw)
+   :format: html
+
+.. list-table::
+   :header-rows: 1
+
+   * - Type
+     - Syntax
+     - Description
+     - Demo
+   * - checkbox
+     - ``<input type="checkbox" name="signUp"/>``
+     - A small box for marking form option as *checked*.
+     - :raw-html:`<label>sign up<input type="checkbox" name="signUp"/></label>`
+
+Examples
+--------
+
+.. admonition:: Example
+
+   One checkbox. No ``value`` attribute is set, so the default value of ``on`` is submitted.
+
+   .. sourcecode:: html
+
+       <label>crew<input type="checkbox" name="crewReady"/></label>
+
+   **Submitted** (if checked)
+
+   ::
+
+      crewReady=on
+
+   `Run it at repl.it <https://repl.it/@launchcode/checkbox-inputs-example>`__
```

Comment:
All the examples use the same code, its a little confusing to have the same link at line 39, 58, 77.  Maybe just keep the one at 77?

Reply:
i will have to come back to this.  not able to change ANY of the links even if i replace urls.  these still provide a working example, even if slightly redundant.  perhaps after updating my current sphinx, this will be an option?

Reply:
still no go.

Reply:
going to keep this as it is right now, but will just have the link at 77.  thanks!